### PR TITLE
[openwrt] Fixed TypeError caused by type mismatch

### PR DESCRIPTION
**Pull request by AnonGuy**
This should resolve issue #118.

I've fixed a TypeError that was raised due to a concatenation (`ipv4 + ipv6`) between two addresses that were of different types, `list` and `str`. I resolved this by applying the same type conversion that was applied to the `ipv4` variable: If only one address is found, convert it to a list of a single element.

After looking through the error traceback that was provided in the issue, it didn't take long to find out what the problem was caused by: When only one ipv6 address is found, the type of the `ipv6` variable is `str`. However, later in the method, the variable is added to the `ipv4` element:
```python
for address in ipv4 + ipv6:
```
`ipv4` will always be a list, as shown by the type conversion that was applied before mine:
```python
if not isinstance(ipv4, list):
    netmask = interface.pop('netmask', 32)
    parsed_ip = self.__netjson_parse_ip(ipv4, netmask)
    ipv4 = [parsed_ip] if parsed_ip else []
```
As far as I understand it, the same logic can be used for the conversion for `ipv6`, by changing the `32` to `128` (there are 128 bits to an ipv6 address, as opposed to the 32 assigned for an ipv4 address):
```python
if not isinstance(ipv6, list):
    netmask = interface.pop('netmask', 128)
    parsed_ip = self.__netjson_parse_ip(ipv6, netmask)
    ipv6 = [parsed_ip] if parsed_ip else []
```
Let me know if there's a flaw in my logic, but this is what I have implemented for the solution to this issue.

### CODE REVIEW

**Comment by AnonGuy** [comment]
I'm currently at college, I'll fix the Travis build and squash commits when I'm at a PC.

**Comment by AnonGuy** [comment]
The Travis builds have been fixed. I'll squash these commits as soon as possible.

File: netjsonconfig/backends/openvpn/parser.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 from ...utils import sorted_dict
 from ..base.parser import BaseParser
 
-vpn_pattern = re.compile('^# openvpn config:\s', flags=re.MULTILINE)
-config_pattern = re.compile('^([^\s]*) ?(.*)$')
+vpn_pattern = re.compile(r'^# openvpn config:\s', flags=re.MULTILINE)
+config_pattern = re.compile(r'^([^\s]*) ?(.*)$')
```
Comment (nemesisdesign) [inline netjsonconfig/backends/openvpn/parser.py]: try not to mix changes that are unrelated to the problem you are fixing, or if you do, at least commit them separately with a commit message that indicates the change is a minor cleanup. For this time it's ok, keep in mind for the next patches.

Comment (AnonGuy) [reply netjsonconfig/backends/openvpn/parser.py]: Okay, I'll keep that in mind! I've added this because it was causing errors in the build, but next time I'll make a separate PR for it.

**Comment by nemesisdesign** [comment]
Implemented in #124
